src/carousel.tsx

```diff
@@ -311,6 +312,7 @@ const BaseCarousel: React.FC<BaseCarouselProps> = ({
     itemClassName,
     withBullets,
     renderBullets,
+    initialActiveItem,
```

Comment:
if you are reacting to changes of this prop, don't call it `initialActiveItem`. `activeItem` would be a better name because this is not the "initial value", it is the current one.

Reply:
its the initial,, because the carousel will be allowed to change the slide without requiting the prop to change (by swiping), thats where the initial word comes from. IMHO the naming is correct.